confluent_kafka/avro/__init__.py

```diff
@@ -98,22 +98,21 @@ def produce(self, **kwargs):
         value_schema = kwargs.pop('value_schema', self._value_schema)
         topic = kwargs.pop('topic', None)
         if not topic:
-            log.error("Topic name not specified.")
             raise ClientError("Topic name not specified.")
         value = kwargs.pop('value', None)
         key = kwargs.pop('key', None)
         if value:
             if value_schema:
                 value = self._serializer.encode_record_with_schema(topic, value_schema, value)
+                log.debug("Value encoded successfully!")
```

Comment:
I just commented on one of these log occurences in the previous review, but all of them should be removed.